lib/Serverless.js

```diff
@@ -427,6 +426,16 @@ class Serverless {
       }
     }
 
+    if (
+      process.argv.slice(2).includes('-v') &&
```

Comment:
Why not check `options.v` ?

Reply:
Because after parsing `cliOptions`, they have resolved full names of options and aliases are stripped e.g. it's only `verbose: true` and no `v` key available.